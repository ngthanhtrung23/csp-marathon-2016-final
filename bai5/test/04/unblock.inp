......
...123
00.123
44.12.
.566..
.577..
