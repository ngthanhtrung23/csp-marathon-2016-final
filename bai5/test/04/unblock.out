17
1 U 1
2 U 1
3 U 1
4 R 4
5 U 1
6 R 2
7 L 1
1 D 3
0 R 2
5 U 3
0 L 1
1 U 3
4 L 3
6 L 3
1 D 3
2 D 3
0 R 3

